velocity: 1.
car_radius: 0.3
min_turning_radius: 1.
